packages/stripe_ios/ios/Classes/StripePlugin+PayPlugin.swift

```diff
@@ -0,0 +1,122 @@
+import Flutter
+import UIKit
+import Stripe
+import PassKit
+
+
+extension StripePlugin {
```

Comment:
Stripe RN? Yeah maybe it should be a good idea. But it would require a way for them to expose the PKPayment object to the framework layer. On our side, this is now done by the `pay` plugin